default/stringtables/fr.txt

```diff
@@ -889,6 +889,9 @@ L'Univers généré contient des erreurs. Consultez le fichier log pour le déta
 SERVER_TURN_EVENTS_ERRORS
 Les événements de tour scriptés contiennent des erreurs. Consultez le fichier log pour le détail des messages d'erreur. La partie peut continuer, mais le gameplay en sera probablement impacté.
 
+#ERROR_PYTHON_AI_CRASHED
+#Python AI for %1% crashed.
+
```

Comment:
Might omit untranslated entries.  They will default to en.txt and is more noticeable for translators.